spec/openapi.yaml

```diff
@@ -7711,6 +7711,309 @@ paths:
+        429:
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        500:
+          $ref: '#/components/responses/Error500ResponseDefinition'
+
+  /jobs-api/v3/accounts/{accountUid}/custom-fields:
+    get:
+      summary: Find custom fields for specified account
+      description: >-
+        Get all custom fields for specified account.
+      tags:
+        - Jobs
+      operationId: getAccountCustomFields
+
+      parameters:
+        - description: Unique identifier for the corresponding account.
+          in: path
+          name: accountUid
+          required: true
+          schema:
+            type: string
+        - name: searchableOnly
+          description: Get only searchable custom fields.
+          in: query
+          required: false
+          schema:
+            type: boolean
+        - name: enabledOnly
+          description: Get only enabled custom fields.
+          in: query
+          required: false
+          schema:
+            type: boolean
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CustomFields'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: SUCCESS
+                      data:
+                        items:
+                          - fieldUid: ngahq6dgh4ed
+                            type: SELECTBOX
+                            fieldName: field-name
+                            enabled: true
+                            required: true
+                            searchable: true
+                            displayToTranslators: true
+                            options:
+                              - option1
+                              - option2
+                            defaultValue: default-field-name
+                            description: custom field description
+                        totalCount: 1
+          description: OK
+        400:
+          $ref: '#/components/responses/Error400ResponseDefinition'
+        401:
+          $ref: '#/components/responses/Error401ResponseDefinition'
+        404:
+          description: Not found error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error404Response'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: NOT_FOUND_ERROR
+                      errors:
+                        - key: null
+                          message: null
+                          details: null
+
+        429:
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        500:
+          $ref: '#/components/responses/Error500ResponseDefinition'
+
+    post:
+      summary: Create new custom field in account
+      description: >-
+        Creates new custom field in account
```

Comment:
Period please. And another point of consistency - if we use `third person` form of statement - lets use it everywhere. I.e. previous endpoint has description `Get all custom fields for specified account.` then it must be changed to `Gets all custom fields for specified account.` One or another way, but must be consistent